tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -26,10 +26,17 @@ def _check_cpuid_x86(test_microvm, expected_cpu_count, expected_htt):
 
 
 def _check_cpu_features_arm(test_microvm):
-    expected_cpu_features = {
-        "Flags": "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
-        "asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs",
-    }
+    if utils.get_instance_type() == "m6g.metal":
+        expected_cpu_features = {
+            "Flags": "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
```

Comment:
Not a blocker at all, but I am quite curious how these feature flags were arrived at. Any points on requirements and what the constraints are would be  useful retrospectively.

Reply:
These are the CPU features that KVM exposes to the guest. For `m6g.metal` the list is the same on the host and the guest, but this is not the case for all platforms because sometimes KVM disables some features by default.